### DISCUSSION TITLE
Inline-math with mathjax

### DISCUSSION THREAD

**Discussion by andrewerf** [xenocrat/chyrp-lite#264] [Q&A] [answered]
I have mathjax enabled, but it seems to only allow out-of-line rendering (with `$$`). When I try to place equations inline (with `$`) they are rendered like other symbols (i.e. `$\lambda$` verbatim, instead of $\lambda$).

Is there a way to render inline-equations in chyrplite?

**Comment by xenocrat** [comment, marked as answer]
Hello there,

Mathjax doesn't support single $ delimiters by default. As they explain in [their documentation](https://docs.mathjax.org/en/v3.0-latest/input/tex/delimiters.html), this would be very likely to cause false positives in parsed text. Instead you should use `\(` and `\)` for inline rendering - but be aware that if you also have Markdown support enabled (Admin > Settings > Content > Markdown) then you'll need to use `\\(` and `\\)` because Markdown uses backslash as an escape character, so you must type two backslashes to get one as the output.

**Reply by andrewerf** [reply]
Thank you, it works!
